starter-code/server.js

```diff
@@ -92,8 +92,8 @@ app.put('/articles/:id', function(request, response) {
     client.query(
       `UPDATE articles
       SET title = $2, category = $3, "publishedOn" = $4, body = $5
-      WHERE author_id = $1;`,
-      [request.body.author_id, request.body.title, request.body.category, request.body.publishedOn, request.body.body]
+      WHERE article_id = $1;`,
+      [request.params.id, request.body.title, request.body.category, request.body.publishedOn, request.body.body]
```

Comment:
How would the author_id get updated in the author table?  It looks like we're giving them the option of changing that author's name, but it would still be the same record with the same author_id.  If they were putting a new author on an existing article, we'd need a different call that mimicks the first two functions from the post/articles request.